src/store/appSlice.ts

```diff
@@ -0,0 +1,38 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+// import type { Location } from '../types';
+
+interface AppState {
+  searchFieldVisible: boolean;
+  isCelsius: boolean;
+}
+
+const initialState: AppState = {
+  searchFieldVisible: false,
+  isCelsius: true,
+};
+
+const appSlice = createSlice({
+  name: 'app',
+  initialState,
+  reducers: {
```

Comment:
here you can add one more field

selectors: {
  ...your selectors
}

and then export it as you do with actions